vana/chain_manager.py

```diff
@@ -252,13 +275,21 @@ def get_current_block(self) -> int:
         Knowing the current block number is essential for querying real-time data and performing time-sensitive
         operations on the blockchain. It serves as a reference point for network activities and data synchronization.
         """
-        return self.web3.eth.block_number
+        return self.circuit_breaker.call(self._get_current_block)
```

Comment:
I'm probably misunderstanding, but does the circuit breaker give us anything additional that the @retry doesn't?